.vscode/extensions.json

```diff
@@ -0,0 +1,3 @@
+{
+  "recommendations": ["jackolope.lit-analyzer-plugin"]
```

Comment:
I'm not sure this does anything which `ts-lit-plugin` doesn't do

Reply:
Just tested: The VSCode extension shows errors like `<a href=${new URL()"}>Example</a>`. Without, they don't show up.

Reply:
Not sure why but it complains:

```
./components/not-found/element.js

    This tag isn't closed.
    80:  <p>${this.l10n("not-found-f
    no-unclosed-tag


  ✖ 1 problem in 1 file (1 error, 0 warnings)
```